Lesson1 (Sequence containers)/Proskuchenko/lesson2task1/isExpressionValid.cpp

```diff
@@ -0,0 +1,30 @@
+#include <forward_list>
+#include <string>
+
+inline char OpositBracket(const char BRACKET) {
+  if (BRACKET == '(') {return ')';}
+  if (BRACKET == '[') {return ']';}
+  if (BRACKET == '{') {return '}';}
+  return 0;
+}
+
+bool isExpressionValid (const std::string& EXPRESSION) {
+  std::forward_list<char> flist_brackets;
+  const size_t EXPRESSION_SIZE = EXPRESSION.size();
```

Comment:
_not consistent codestyle_
use lower camel case or snake case for automatic duration variables and function arguments